src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -54,13 +54,18 @@ const propTypes = {
 
         /** Whether or not the IOU step is loading (retrieving users preferred currency) */
         loading: PropTypes.bool,
+
+        // Selected Currency Code of the current IOU
```

Comment:
NAB, these propTypes docs should be using `/** */` style comments not the `//`

Reply:
I updated the other propTypes docs as well! Thanks for pointing that out.